docs_user/assemblies/assembly_migrating-the-object-storage-service.adoc

```diff
@@ -5,7 +5,9 @@ ifdef::context[:parent-context: {context}]
 :context: migrate-object-storage-service
 
 = Migrating the {object_storage} to {rhos_long_noacro} nodes
-If you are using the {rhos_prev_long} {object_storage_first_ref} as an Object Storage service, you must migrate your Object Storage service to {rhos_long_noacro} nodes. If you are using the Object Storage API of the Ceph Object Gateway (RGW), you can skip this chapter.
+If you are using the {rhos_prev_long} {object_storage_first_ref} as an Object Storage service, you must migrate your Object Storage service to {rhos_long_noacro} nodes. 
+
+If you are using the Object Storage API of the Ceph Object Gateway (RGW), you can skip this chapter and run Ceph Migration. Even if in case ceph migration is not required , it is mandatory to run the chapters xref:deploying-a-ceph-ingress-daemon_migrating-ceph-rgw[Deploying a Ceph ingress daemon] and xref:updating-the-object-storage-endpoints_migrating-ceph-rgw[Create/Update the Object Storage service endpoints]. 
```

Comment:
`Even if in case ceph migration is not required` can be:

```
If your are not planning to migrate Ceph daemons out of controller nodes, it is required to perform the steps described by chapters ....
```